src/bindings/js/node/tests/unit/basic.test.js

```diff
@@ -345,12 +345,9 @@ describe("ov basic tests.", () => {
     });
 
     it("Test importModelSync(stream, device, config: unsupported property) \
-    throws", () => {
+    no exception core property not passed to device", () => {
       const tmpDir = "/tmp";
-      assert.throws(
-        () => core.importModelSync(userStream, "CPU", { CACHE_DIR: tmpDir }),
-        /Unsupported property CACHE_DIR by CPU plugin./,
-      );
+      core.importModelSync(userStream, "CPU", { CACHE_DIR: tmpDir });
```

Comment:
I would suggest replacing the parameter being tested with another one. In this case, we are clearly passing a parameter that cannot be accepted by the CPU plugin and getting the expected error:
```suggestion
    throws", () => {
      assert.throws(
        () => core.importModelSync(userStream, "CPU", { NPU_DEVICE_TOTAL_MEM_SIZE: 1024}),
        /Unsupported property NPU_DEVICE_TOTAL_MEM_SIZE by CPU plugin./,
      );
    });
```